backend/app/services/work_queue_service.py

```
@@ -465,6 +465,24 @@ def create_queue(self, user_id: int, data: WorkQueueCreate) -> WorkQueueResponse
                 spec["autoProcess"] = data.autoProcess.model_dump()
+                # Validate subscriptionRef if auto-process is enabled
+                if data.autoProcess.enabled and data.autoProcess.subscriptionRef:
+                    ref = data.autoProcess.subscriptionRef
+                    subscription = (
+                        db.query(Kind)
+                        .filter(
+                            Kind.kind == "Subscription",
+                            Kind.namespace == ref.namespace,
+                            Kind.name == ref.name,
+                            Kind.user_id == ref.userId,
+                            Kind.is_active == True,
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)